taken from upstream git

From 7365eac2ee69125eed33d08a2a179dc99f201090 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 8 Nov 2011 16:56:03 -0500
Subject: [PATCH] add libdir/exec_prefix to xcb-xproto.pc

If we want to base other paths off of exec_prefix or libdir (like the
python path), we can't because the xcb-proto.pc.in file omits those
variables.  Add them so people can leverage them in their own builds.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 xcb-proto.pc.in |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/xcb-proto.pc.in b/xcb-proto.pc.in
index 6cbcf27..d312e6e 100644
--- a/xcb-proto.pc.in
+++ b/xcb-proto.pc.in
@@ -1,6 +1,8 @@
 prefix=@prefix@
+exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
 datadir=@datadir@
+libdir=@libdir@
 xcbincludedir=@xcbincludedir@
 pythondir=@pythondir@
 
-- 
1.7.6.1

